src/chrome/content/rules/Marx.org.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+	Time out:
+		marx.org
+		www.marx.org
+
+-->
+<ruleset name="Marx.org">
+
+	<target host="marx.org" />
+	<target host="www.marx.org" />
+
+	<rule from="^http://(www\.)?marx\.org/"
+		to="https://www.marxists.org/" />
+
+</ruleset>
```

Comment:
Is it possible to remove this redirect?

Reply:
I've removed this ruleset entirely as the redirect wasn't correct. See 17f3362